docs/building-a-form/available-components.md

```diff
@@ -326,7 +326,43 @@ You can set follow up questions to appear only if the user answers a form questi
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+There are 2 fields you can use to conditionally expand a form field:
```

Comment:
It might be helpful to contrast `expandUnder` with [`depends`](https://github.com/usds/us-forms-system/blob/master/docs/building-a-form/common-patterns-for-building-forms.md#conditionally-including-a-page) (and vice-versa) since they're accomplishing similar things. They seem similar enough that someone might expect to find them near each other in the docs.